docs/content/drafts/ActionList2.mdx

```diff
@@ -373,3 +472,13 @@ render(<SelectFields />)
     hasFigmaComponent: true
   }}
 />
+
+## Further reading
+
+- [Interface guidelines: Action List](https://primer.style/design/components/action-list)
+
+## Related components
+
+- [ActionMenu](/drafts/ActionMenu2)
```

Comment:
Should we call this "ActionMenu2" in the link text?

Reply:
I'll leave that up to @siddharthkp